test/reserveSetup.js

```diff
@@ -73,7 +73,7 @@ module.exports.setupConversionRateV1 = async function(tokens, admin, operator, a
     await convRatesInst.setBaseRate(tokenAdd, baseBuyRate, baseSellRate, buys, sells, currentBlock, indices, {from: operator});
 
     //set compact data
-    compactBuyArr = [0, 0, 0, 0, 0, 06, 07, 08, 09, 1, 0, 11, 12, 13, 14];
+    compactBuyArr = [0, 0, 0, 0, 0, 6, 7, 8, 9, 10, 11, 12, 13, 14];
```

Comment:
one value removed here?

Reply:
Yes, extra value lol. Max array should be 14, but it's 15. Hardhat caught this bug thanks to their stricter input validation checks